src/lib/gql/GqlHasuraAdmin.ts

```diff
@@ -520,4 +543,59 @@ export class Gql {
     });
     return profiles.length > 0;
   }
+
+  async insertUser(address: string, name: string, circleId: number) {
+    const { insert_users_one } = await this.q('mutation')({
+      insert_users_one: [
+        {
+          object: {
+            address: address,
+            circle_id: circleId,
+            name: name,
+          },
+        },
+        {
+          id: true,
+        },
+      ],
+    });
+    return insert_users_one;
+  }
+
+  async insertVouch(nomineeId: number, voucherId: number) {
+    const { insert_vouches_one } = await this.q('mutation')({
+      insert_vouches_one: [
+        {
+          object: {
+            nominee_id: nomineeId,
+            voucher_id: voucherId,
+          },
+        },
+        {
+          id: true,
```

Comment:
we can return the `nominee` object here so you wouldn't need to refetch after every insert